README.md

```diff
@@ -53,7 +53,9 @@ docker run --net=host -d -e CONFIGPROXY_AUTH_TOKEN=$TOKEN \
 Usage: orchestrate.py [OPTIONS]
 
 Options:
-
+  --allow_origin                   Set the Access-Control-Allow-Origin header.
```

Comment:
This works with `--allow-origin` too right? I'd rather see the `-` form for what users see.